keyboards/mt84/readme.md

```diff
@@ -0,0 +1,21 @@
+# Mt84rgb 
+
+![Mt84rgb](https://i.imgur.com/2fJg3DDl.png)
+
+A 84% keyboard PCB made by MT.
+It supports standard 84-key layout, with hotswap sockets and per-key RGB backlight.
+
+
+* Keyboard Maintainer: [MT](https://github.com/704340378)
+* Hardware Supported: mt84rgb (atmega32u4)
+* Hardware Availability: [麦田外设](https://shop110310565.taobao.com)
+
+Make example for this keyboard (after setting up your build environment):
+
+    make mt84:default
+    
+Flashing example for this keyboard:
+
+    make mt84:default:flash
+

```

Comment:
Can you add "clear instructions on how to reset the board into bootloader mode" as per the [PR checklist](https://docs.qmk.fm/#/pr_checklist?id=keyboard-prs)?

Reply:
press the reset key will let it into bootloader.
sorry i dont know what line should i add in the readme file ,i had check other keyboads and no find this instructions in there readme.md
can you give the line i will add it in ,thank you very much

Reply:
thank you very much